client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -311,6 +311,7 @@ export default function WPCheckout( {
 	}
 
 	const isDIFMInCart = hasDIFMProduct( responseCart );
+	const showUpsellMonthly = responseCart?.products?.some( isMonthlyProduct );
```

Comment:
While it does the right thing I think this variable name might be a little misleading. Maybe something like `hasMonthlyProduct`?

Reply:
I agree with this, `hasMonthlyProduct ` looks better.